requirements.txt

```diff
@@ -1,11 +1,15 @@
 asgiref==3.3.4
+dj-database-url==0.5.0
 Django==3.2
 django-cors-headers==3.7.0
+django-heroku==0.3.1
 djangorestframework==3.12.4
 djangorestframework-simplejwt==4.6.0
 gunicorn==20.1.0
 Pillow==8.2.0
+psycopg2==2.8.6
```

Comment:
Any reason we are using `psycopg2`? Are we writing RAW SQL commands Postgres database anywhere?

Reply:
Its a connector to connect postgres db to django